diff --git a/node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/SvgPackage.java b/node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/SvgPackage.java
index ce96152..2bed44c 100644
--- a/node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/SvgPackage.java
+++ b/node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/SvgPackage.java
@@ -239,9 +239,10 @@ public class SvgPackage extends TurboReactPackage implements ViewManagerOnDemand
   }
 
   /** {@inheritDoc} */
+  @Nullable
   @Override
   public List<String> getViewManagerNames(ReactApplicationContext reactContext) {
-    return (List<String>) getViewManagersMap(reactContext).keySet();
+      return new ArrayList<>(mViewManagers.keySet()); //anton mikhailouski fix
   }
 
   @Override
diff --git a/node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/VirtualView.java b/node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/VirtualView.java
index 4fa8417..e8b28d3 100644
--- a/node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/VirtualView.java
+++ b/node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/VirtualView.java
@@ -103,7 +103,8 @@ public abstract class VirtualView extends ReactViewGroup {
   ArrayList<PathElement> elements;
   PointerEvents mPointerEvents;
 
-  void setPointerEvents(PointerEvents pointerEvents) {
+  @Override
+  public void setPointerEvents(PointerEvents pointerEvents) { //anton mikhailouski fix
     mPointerEvents = pointerEvents;
   }
 
